### ISSUE TITLE
why no BatchNorm layer?

### ISSUE THREAD

**Issue by joyousrabbit** [shihenw/convolutional-pose-machines-release#38]
Hello Wei,
Great work. But why didn't you use BatchNorm layer which is said very good for speed and performence?

Hope Reply.

**Comment by hyqneuron** [comment]
Hi Shih-En, since we're on the topic of training speed, may I ask here how many GPUs did you use to train your models?

**Comment by joyousrabbit** [comment]
@hyqneuron 2 1070 5 days.

**Comment by shihenw** [comment]
We've tried adding BN layers after each CONV layers, but didn't get better convergence speed and accuracy. But there's a big speedup with VGG-pretrained model vs. training from scratch.

Given the VGG-pretrained model, I was able to get the result within 3 days with 2 Titan X's (pascal).

**Comment by joyousrabbit** [comment]
Thank you for info.
I tried also BN  for another project, and it got the same observation as
yours: no speedup, no amelioration.

2016-12-30 2:01 GMT+01:00 Shih-En Wei <notifications@github.com>:
